code/game/objects/structures/crates_lockers/closets/secure/personal.dm

```diff
@@ -33,8 +30,6 @@
 	icon_opened = "cabinetdetective_open"
 	icon_broken = "cabinetdetective_broken"
 	icon_off = "cabinetdetective_broken"
-	burn_state = FLAMMABLE

```

Comment:
Oh, I know what happened, I synced to the latest version of the git but I had the old file still opened, so i was editing the old file.

Moot now though.